pkgs/servers/monitoring/grafana/plugins/grafana-oncall-app/default.nix

```
@@ -0,0 +1,19 @@
+{ grafanaPlugin, lib }:
+
+grafanaPlugin rec {
+  pname = "grafana-oncall-app";
+  version = "1.3.106";
+  zipHash = {
+    x86_64-linux = "sha256-8TL8HqiYCUxY2tCItpavYb77rGIpcJBWexwykFNWONg=";
+    aarch64-linux = "sha256-4rCj+NaKPZbuVohlKmSf1M6m5ng9HZMrwzBCgLPdiok=";
+    x86_64-darwin = "sha256-bpey6EwwAqXgxjvjJ6ou4rimidHCpUr+Z89YpAZK7z8=";
+    aarch64-darwin = "sha256-u/U2lu4szf9JFt/zfhGmWKF2OUqpJDNaSI69EDdi1+w=";
+  };
+  meta = with lib; {
+    description =
+      "Collect and analyze alerts, escalate based on schedules and deliver them to Slack, Phone Calls, SMS and others.";
```

> `meta.description` must:
> - Be short, just one sentence.
> - [...]
> - Not end with a period (or any punctuation for that matter).

https://github.com/NixOS/nixpkgs/tree/c10fb0df312867b0ae47a580818b317a9ae61f21/pkgs#meta-attributes

How about (taking the beginning of the official GitHub repo description):

```suggestion
    description = "Developer-friendly incident response for Grafana";
```